src/cnaas_nms/db/helper.py

```diff
@@ -18,57 +18,34 @@ def canonical_mac(mac):
     return str(na_mac)
 
 
-def find_mgmtdomain(session, hostnames: List[str]) -> Optional[Mgmtdomain]:
-    """Find the corresponding management domain for a pair of
-    distribution switches.
+def find_mgmtdomain_one_device(session, device0: Device) -> Optional[Mgmtdomain]:
```

Comment:
The goal was to lower the complexity and length per function so it's somewhat easier to understand when reading the code. I think it would be good a good practice overall, as we often have very long function which makes it difficult to understand. I think sonarcloud has some limits set up about complexity per function of max 25.